### DISCUSSION TITLE
Logo Showcase

### DISCUSSION THREAD

**Discussion by loftwah** [EddieHubCommunity/LinkFree#6799] [Ideas]
Professionals who have worked with notable brands or clients can highlight their experience.

Users could also charge companies to display their logo here.

**Comment by poorvijn** [comment]
can we create a separate tab called 'Experience/Work' where logos of brands\clients the user has worked for can be displayed, and when someone hovers over each brand or clicks on them, maybe a testimonial from that brand pops up. It would be like just like a testimonial section but for brands/clients
